apis/r/src/Makevars.in

```diff
@@ -1,7 +1,7 @@
 CXX_STD = CXX17
 
 ## We need the TileDB Headers, and for macOS aka Darwin need to set minimum version 10.14 for macOS
-PKG_CPPFLAGS = -I. -I../inst/include/ @tiledb_include@ -DR_BUILD @cxx17_macos@ -Iinclude
+PKG_CPPFLAGS = -I. -I../inst/include/ -Ilibtiledbsoma/external/include @tiledb_include@ -DR_BUILD @cxx17_macos@ -Iinclude
```

Comment:
There is a large parallel pending change that removes all this when we switch to a shared libtiledbsoma library.

Bigger picture the -I location should not be hardwired by learned in `configure` probably from `pkg-config`